src/hooks/useIsDapp.ts

```diff
@@ -0,0 +1,16 @@
+import { usePathname } from 'next/navigation';
+import { JUMPER_LEARN_PATH, JUMPER_LOYALTY_PATH } from 'src/const/urls';
+
+const checkDappPath = (path: string | null) => {
+  if (path) {
+    return (
+      !path.includes(JUMPER_LEARN_PATH) && !path.includes(JUMPER_LOYALTY_PATH)
+    );
+  }
+};
+
+export const useIsDapp = () => {
```

Comment:
Very much against this, what is the goal you want to achieve? If you want to know if you are on a "widget" page, you can get the info from `[lng]/(main)/layout.tsx` directly